docs/network-flow-visibility.md

```diff
@@ -602,6 +602,27 @@ use the checked-in [deployment yaml](/build/yamls/flow-visibility.yml):
 kubectl apply -f https://raw.githubusercontent.com/antrea-io/antrea/main/build/yamls/flow-visibility.yml
 ```
 
+To be noticed, Grafana is exposed as a NodePort Service by default in `flow-visibility.yml`.
+If the given K8s cluster support LoadBalancer Services, Grafana can be changed to a
```

Comment:
s/support/supports

Reply:
"Grafana can be exposed through a LoadBalancer Service..."?